.github/workflows/gradle-cache-check.yml

```diff
@@ -0,0 +1,48 @@
+name: Periodic Gradle Build Cache Check
+on:
+  schedule:
+    - cron: '0 01 * * 1'
+
+env:
+  LC_ALL: "en_US.UTF-8"
+  GRADLE_ENTERPRISE_ACCESS_KEY: ${{ secrets.GRADLE_ENTERPRISE_ACCESS_KEY }}
+
+jobs:
+  build-cache-check:
+    if: github.repository == 'line/armeria'
+    runs-on: self-hosted
+    steps:
+      - id: setup-jdk-19
+        name: Set up JDK 19
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: '19'
+
+      - name: Create working directory
+        run: mkdir scripts
+
+      - name: Download script
+        working-directory: scripts
+        run: curl -s -L -O https://github.com/gradle/gradle-enterprise-build-validation-scripts/releases/download/v2.5/gradle-enterprise-gradle-build-validation-2.5.zip
+
+      - name: Unzip script
+        working-directory: scripts
+        run: unzip -q -o gradle-enterprise-gradle-build-validation-2.5.zip
+
+      - name: Run script
+        working-directory: scripts/gradle-enterprise-gradle-build-validation
+        run: |
+          ./03-validate-local-build-caching-different-locations.sh -f \
+          -r https://github.com/line/armeria.git -t build \
+          -a --parallel\ -Pretry=true\ -PfailOnPassedAfterRetry=false
```

Comment:
```suggestion
          -a --parallel -Pretry=true -PfailOnPassedAfterRetry=false
```

Reply:
This is actually intentional, just tried a run in my local:

```
user@AL02437565 gradle-enterprise-gradle-build-validation % ./03-validate-local-build-caching-different-locations.sh -f \
          -r https://github.com/line/armeria.git -t buildWeb \
          -a --parallel -Pretry=true -PfailOnPassedAfterRetry=false
FATAL ERROR: Got an unexpected argument '-Pretry=true'
```

```
user@AL02437565 gradle-enterprise-gradle-build-validation % ./03-validate-local-build-caching-different-locations.sh -f \
          -r https://github.com/line/armeria.git -t buildWeb \
          -a --parallel\ -Pretry=true\ -PfailOnPassedAfterRetry=false

Cloning armeria
```

Reply:
Oops, sorry about it. 😓